src/test/resources/application.yml

```diff
@@ -0,0 +1,79 @@
+#
```

Comment:
What is the need to duplicate the whole file? Can only required properties by added to this file?

Reply:
I can remove this file